Vero-Ventures/llm-swarm
# Add updated Jupyter Notebooks for LLM testing

## Summary
This is a Jupyter Notebook with some basic code to load a Llama3 model; create some agents, tasks, and a crew; read from a file containing functions; then write the updated functions to another file.

This pipeline is not yet finished.

## Testing

** I tried running this without Ollama running. It did not work. For those on Windows and Linux, please try running this twice -- once with Ollama running, once without. **

1. Checkout the `rhys/research_update` branch
2. Open the integrated terminal in VSCode
3. Run `poetry run jupyter notebook`
4. Navigate to `rhys_research > LLM_Test`
5. At the top of the screen, click "Kernel", then "Restart Kernel and Run All Cells..."
6. Once processing ends, check the `test_outputs` folder and see if a file was written.
